File: Day 1/Java/LargestNumber.java
package sample;
import java.util.Scanner;
public class LargestNumber {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		System.out.println("Enter three numbers: ");
		int n1=s.nextInt();
		int n2=s.nextInt();
		int n3=s.nextInt();
		if(n1>n2) {
			if(n2>n3) {
				System.out.println(n1+" is the largest");
			}
			else if(n3>n1) {
				System.out.println(n3+" is the largest");
			}
		}
		else if(n2>n3){
			System.out.println(n2+" is the largest");
		}
		else {
			System.out.println(n3+" is the largest");
		}
	}
}

File: Day 1/Python/LargestNumber.py
n1=int(input("Enter number 1: "))
n2=int(input("Enter number 2: "))
n3=int(input("Enter number 3: "))
if n1>n2:
    if n2>n3:
        print(f'{n1} is the largest')
    elif n3>n1:
        print(f'{n3} is the largest')
elif n2>n3:
    print(f'{n2} is the largest')
else:
    print(f'{n3} is the largest')